src/tests/unit/components/TextEditor.spec.js

```diff
@@ -106,7 +107,7 @@ describe('TextEditor', () => {
 
 			const editor = await VirtualTestEditor.create({
 				initialData: '<p>bonjour bonjour</p>',
-				plugins: [ParagraphPlugin],
+				plugins: [ParagraphPlugin, MailPlugin],
```

Comment:
Actually the test is not helpful anymore as the behavior "add margin:0 to every pargraph" is now done by the plugin and therefor not bound to onEditorReady.